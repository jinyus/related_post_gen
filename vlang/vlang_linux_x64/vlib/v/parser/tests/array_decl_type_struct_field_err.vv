
struct Config {
mut:
	max_size          string
	filter_until_hour int   = 240 // 10 days
	prune_list        array = ['all']
	time_interval     int   = 300 // 5 min
}
