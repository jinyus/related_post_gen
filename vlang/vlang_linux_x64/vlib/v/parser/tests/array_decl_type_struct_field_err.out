vlib/v/parser/tests/array_decl_type_struct_field_err.vv:6:20: error: `array` is an internal type, it cannot be used directly. Use `[]int`, `[]Foo` etc
    4 |     max_size          string
    5 |     filter_until_hour int   = 240 // 10 days
    6 |     prune_list        array = ['all']
      |                       ~~~~~
    7 |     time_interval     int   = 300 // 5 min
    8 | }
